packages/shared/src/lib/user.ts

```diff
@@ -24,12 +24,13 @@ export interface PublicProfile {
   premium: boolean;
   image: string;
   reputation: number;
+  permalink: string;
 }
 
 export interface UserProfile {
   name: string;
   email: string;
-  username?: string;
+  username: string;
```

Comment:
In rare cases, this can be undefined. We will fix it once we work on the auth system

Reply:
I'm super excited about our auth system!